pkg/api/repositories.go

```diff
@@ -20,6 +20,7 @@ type RepositoryResponse struct {
 	MetadataVerification         bool     `json:"metadata_verification"`               // Verify packages
 	RepositoryUUID               string   `json:"-" swaggerignore:"true"`              // UUID of the dao.Repository
 	Snapshot                     bool     `json:"snapshot"`                            // Enable snapshotting and hosting of this repository
+	LastSnapshotTaskUUID         string   `json:"last_snapshot_task_uuid,omitempty"`   // UUID of the last snapshot task
```

Comment:
We don't use omitempty on the other fields. Do you prefer it be used for snapshots specifically?

Reply:
For snapshot related things, I suppose it could be an empty string. I've done something similar [here](https://github.com/content-services/content-sources-backend/pull/342/files#diff-837db424b3bce5a001d8266958327c228eb1a97df5f7c23cf24cf55f52c1b6d5R26)

Reply:
I was mostly just curious about it. I could see omitting fields making sense because some repositories will not have snapshotting enabled.